apps/api/src/app/layouts/usecases/set-default-layout/set-default-layout.use-case.ts

```diff
@@ -13,7 +13,7 @@ export class SetDefaultLayoutUseCase {
     const defaultLayoutId = await this.findDefaultLayoutId(command.environmentId, command.organizationId);
 
     try {
-      if (defaultLayoutId) {
+      if (defaultLayoutId && defaultLayoutId !== command.layoutId) {
```

Comment:
This condition should be out of the try/catch. If the defaultLayoutId is the same as the one wanted to be set as default, it is not needed to execute anything.

Reply:
Good catch.